cibyl/models/ci/build.py

```diff
@@ -53,7 +53,7 @@ def __str__(self, indent=0, verbosity=0):
             build_str += Colors.blue(f"\n{indent_space}  Status: ")
             if self.status.value == "SUCCESS":
                 build_str += Colors.green(f"{self.status.value}")
-            if self.status.value == "FAILURE":
+            if self.status.value in ("FAILURE", "UNSTABLE"):
```

Comment:
Jenkins associates unstable jobs with yellow. I think we should do the same.

Reply:
Oh, I didn't realize that case using elasticsearch. I'll change it, thank u.

Reply:
Done.